go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/cs3org/go-cs3apis v0.0.0-20210802070913-970eec344e59
 	github.com/eventials/go-tus v0.0.0-20200718001131-45c7ec8f5d59
 	github.com/gdexlab/go-render v1.0.1
+	github.com/go-chi/chi v1.5.4
```

Comment:
could you fetch the latest version of Chi?

`go get github.com/go-chi/chi/v5` should do it

Reply:
Done. Which makes me wonder.  Should we update ocis to chi/v5 as well? It's at some v4 release currently.

Reply:
Yes, we [should update it as well](https://github.com/owncloud/ocis/blob/master/go.mod#L26), good catch

Reply:
Yes, we should do that but we would need to update the protobuf plugins first, they are still using the old go-chi version.
I already tried. :see_no_evil:

Reply:
Done -> https://github.com/owncloud/ocis/pull/2429